rcap/http_rulefilter_test.go

```diff
@@ -187,6 +187,89 @@ func TestBlockedDomains(t *testing.T) {
 	})
 }
 
+func TestAllowedPorts(t *testing.T) {
```

Comment:
This test, as well as the others, can be turned into table driven test case. I find they're more uniform and more maintainable and clearer on what's being tested, and deduplicates code.

Reply:
Agreed, though this should not be the PR to do it. 

Opened https://github.com/suborbital/reactr/issues/211 to track this request

Reply:
Thank you! Updated to approved :)